src/utils.rs

```diff
@@ -8,7 +8,6 @@ cfg_if! {
     // For more details see
     // https://github.com/rustwasm/console_error_panic_hook#readme
     if #[cfg(feature = "console_error_panic_hook")] {
-        extern crate console_error_panic_hook;
         pub use self::console_error_panic_hook::set_once as set_panic_hook;
```

Comment:
Does this need to not use `self::` now?

Reply:
Good catch, I filed https://github.com/rustwasm/wasm-pack-template/issues/39 about CI not catching this